pom.xml

```diff
@@ -13,23 +13,25 @@
   <parent>
     <groupId>io.takari</groupId>
     <artifactId>takari</artifactId>
-    <version>25</version>
+    <version>28</version>
   </parent>
 
   <groupId>io.takari.maven</groupId>
   <artifactId>concurrent-build-logger</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>${revision}</version>
```

Comment:
No, don't parametrize project version, artifacts deployed to repositories should have have literal version in their pom.

Reply:
Fixed